bokeh/models/tools.py

```diff
@@ -138,35 +133,20 @@ class Toolbar(ToolbarBase):
     Specify a tap/click tool to be active when the plot is displayed.
     """)
 
+class ProxyToolbar(ToolbarBase):
+    ''' A toolbar that allow to merge and proxy tools of toolbars in multiple plots. '''
 
-class ToolbarBox(Box):
+class ToolbarBox(LayoutDOM):
     ''' A layoutable toolbar that can accept the tools of multiple plots, and
     can merge the tools into a single button for convenience.
 
     '''
-    def _check_empty_layout(self):
-        # Overriding the children check from Box. As toolbarbox's children
-        # are normally set JS side.
-        return None
-
-    toolbar_location = Enum(Location, default='right', help="""
-        Should the toolbar be presented as if it was stuck to the `above`, `right`, `left`, `below`
-        edge of a plot. Default is `right`.
-    """)
 
-    tools = List(Instance(Tool), help="""
-    A list of tools to add to the plot.
-    """)
-
-    merge_tools = Bool(default=True, help="""
```

Comment:
Also need a release note about this if we are removing it. How does a user control merged vs non-merged intent?

Reply:
This actually was never used. There is still `merge_tools` option to `gridplot()` which governs whether a grid plot will have a shared toolbar or not.

Reply:
OK